### ISSUE TITLE
AttributeError: PyinotifyLogger instance has no attribute 'trace'

### ISSUE THREAD

**Issue by cedwards** [HubbleStack/Nova#204]
Nova fails if Pulsar is active with the following error:

```
[root@cst1.dmz ~]# salt minion.site.domain.tld hubble.audit
minion.site.domain.tld:
    The minion function caused an exception: Traceback (most recent call last):
      File "/usr/lib/python2.6/site-packages/salt/minion.py", line 1078, in _thread_return
        return_data = func(*args, **kwargs)
      File "/var/cache/salt/minion/extmods/modules/hubble.py", line 96, in audit
        show_compliance=show_compliance)
      File "/var/cache/salt/minion/extmods/modules/hubble.py", line 346, in top
        called_from_top=True)
      File "/var/cache/salt/minion/extmods/modules/hubble.py", line 143, in audit
        log.trace('hubble.py configs:')
    AttributeError: PyinotifyLogger instance has no attribute 'trace'
```

This only seems to affect my CentOS 6 systems.

```
python-inotify-0.9.1-1.el6.noarch
python-inotify-0.9.4-4.el7.noarch
```

Note: this only fails when the master sends the command. Replicating the issue with `salt-call` doesn't seem to work.

**Comment by basepi** [comment]
This is caused by https://github.com/seb-m/pyinotify/issues/26 and is fixed in pyinotify 0.9.6.

I'm thinking we need to find a way to get a newer version, rather than trying to work around this. But I'll ping you on slack.

**Comment by basepi** [comment]
Fixed in #211
